contents/docs/cdp/batch-exports/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Batch exports
+title: Analytics batch exports
```

Comment:
Maybe this is something to debate, but I always think of analytics as the "analysis of data", not the data itself, making this be more confusing than just leaving it how it is.

I'm also not sure what Google keywords we are trying to rank for. Batch exports isn't a top of funnel product, I don't see people searching this.

Reply:
In these cases, it's less about the ranking and more about how the headline reads at the SERP. A bunch of these look pretty spare from outside.

"Batch exports" of what? The lack of context kinda sucks.

vs.

"oh, batch exports of analytics data"

Should it be "Analytics data batch exports"?

Reply:
Or even "Batch export analytics data"

Reply:
But no one is going to see this page on the SERP, I feel like they will find this page through website navigation, which will show batch exports + the destination name. 

Saying "Analytics data batch exports" reads to me as implying we have another type of batch exports.

Reply:
> But no one is going to see this page on the SERP

WELL: if you're like "hmm, I want to do a data export from PostHog," you're going to search `posthog data export`. This page, appropriately, ranks #1 there, and it looks like this:

<img width="756" alt="Screenshot 2024-11-05 at 2 28 41 PM" src="https://github.com/user-attachments/assets/291c2237-1e7a-42c6-9fb5-0eae6390b0fc">

…which makes me want to revert all the export changes, I think that looks great as is.